script/download-node.js

```diff
@@ -37,8 +37,14 @@ var downloadTarballAndExtract = function(url, location, callback) {
   });
   stream.on('end', callback.bind(this, tempPath));
   stream.on('error', callback);
-  request.createReadStream({url: url}, function(requestStream) {
-    requestStream.pipe(zlib.createGunzip()).pipe(stream);
+  request.createReadStream({url: url}, function(res) {
```

Comment:
Mind keeping the variable as `requestStream`, having both `res` and `resp` variables is a little confusing.